src/main/java/core/basesyntax/model/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Bulldozer, You can do it");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Ohh Bulldozer, try again");
```

Comment:
Similarly, the message 'Ohh Bulldozer, try again' does not clearly indicate that the Bulldozer has stopped working. A more informative message would be 'Bulldozer has stopped working.'